### DISCUSSION TITLE
"Unable to verify macOS" when attempting to update to macOS 13.5.2 (version 30.2)

### DISCUSSION THREAD

**Discussion by macinthass** [grahampugh/erase-install#434] [General]
I have seen a number of Macs stuck during the update preparation for this reason "Unable to verify macOS." Any ideas? Using the latest version 30.2 and trying to update to macOS 13.5.2 which was released today.

**Comment by GabeShack** [comment]
We are seeing on 13.5.2 that it hangs on 98% at the screen "Preparing MacOS installer".  This is only affecting M1 and apple silicone machines though.
